homeassistant/components/number/__init__.py

```diff
@@ -190,7 +190,7 @@ class NumberEntity(Entity):
     _attr_native_max_value: float
     _attr_native_min_value: float
     _attr_native_step: float
-    _attr_native_value: float
+    _attr_native_value: float | None
```

Comment:
The property is defined as `float | None`, so why would the attribute not be allowed to return None?